src/api/open-service.ts

```diff
@@ -72,38 +72,26 @@ export default function openService(
           // eslint-disable-next-line no-case-declarations
           const { entityKey } = options as ServiceConnectionType;
 
-          openWindow(
-            `${VAULT_DOMAINS[mode]}/service/${entityKey}?${getQueryValue()}`,
-            getIsTabValue(isNewTab)
-          );
+          openWindow(`${VAULT_DOMAINS[mode]}/service/${entityKey}?${getQueryValue()}`, getIsTabValue(isNewTab));
           break;
 
         case 'connection-setting':
           // eslint-disable-next-line no-case-declarations
           const { credentialId } = options as ConnectionSettingType;
 
-          openWindow(
-            `${VAULT_DOMAINS[mode]}/connection/${credentialId}?${getQueryValue()}`,
-            getIsTabValue(isNewTab)
-          );
+          openWindow(`${VAULT_DOMAINS[mode]}/connection/${credentialId}?${getQueryValue()}`, getIsTabValue(isNewTab));
           break;
 
         case 'customer-support':
         default:
-          openWindow(
-            `${VAULT_DOMAINS[mode]}/customer-support?${getQueryValue()}`,
-            getIsTabValue(isNewTab)
-          );
+          openWindow(`${VAULT_DOMAINS[mode]}/customer-support?${getQueryValue()}`, getIsTabValue(isNewTab));
           break;
       }
 
       break;
 
     case 'myaccount-settings':
```

Comment:
Since this is V3, I think we can change this to `myaccount` and change its parameters to be the same as login-link, this way it is not just limited to `settings`.

Reply:
Fixed.